### PR TITLE
Fix upgrading guide.

### PR DESCRIPTION
**Pull request by omazin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ last released version intended for Terraform 0.11.x is
 
 The current version is 2.X. The following guides are available to assist with upgrades:
```
Comment (aaron-lane) [inline README.md]: This statement should be fixed as well.
```suggestion
The current version is 1.X. The following guides are available to assist with upgrades:
```

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ last released version intended for Terraform 0.11.x is
 
 The current version is 2.X. The following guides are available to assist with upgrades:
 
-- [1.X -> 2.0](./docs/upgrading_to_gsuite_export_v2.0.md)
+- [1.X -> 2.0](./docs/upgrading_to_gsuite_export_v1.0.md)
```
Comment (aaron-lane) [inline README.md]:
```suggestion
- [0.X -> 1.0](./docs/upgrading_to_gsuite_export_v1.0.md)
```

**Review by aaron-lane — CHANGES_REQUESTED** [review]
Thanks @omazin!

File: README.md (hunk 2 above)
Comment (omazin) [reply README.md]: good catch

**Review by aaron-lane — APPROVED** [review]

**Pull request merged**